tests/SitemapTest.php

```diff
@@ -75,4 +75,16 @@ public function testPriorityValidation()
 
         unlink($fileName);
     }
+
+    public function testLocationValidation()
+    {
+        $this->setExpectedException('InvalidArgumentException');
```

Comment:
If the expected message is changed need rewrite test. This is logical.
Test exception class name often is important because allows you not to be confused with other exception (suddenly emerged)